content/ember/v3/use-notify-prop-change.md

```diff
@@ -33,3 +33,7 @@ Ember.notifyPropertyChange(object, 'someProperty');
 doStuff(object);
 object.notifyPropertyChange('someProperty');
 ```
+
+If you are an addon author and need to support both you can use the polyfill
```

Comment:
Slight tweak here: you should mention “both” of what...

Reply:
Updated to, hopefully, be more clear.